src/schemas/json/function.json

```diff
@@ -514,6 +516,59 @@
           "description": "The text of the email."
         }
       }
+    },
+    "sqlBinding": {
+      "properties": {
+        "connectionStringSetting": {
```

Comment:
Noticed that other bindings in this file are using below format for connection string. We can use the same for consistency.

```json
        "connection": {
          "type": "string",
          "description": "An app setting (or environment variable) with the storage connection string to be used by this binding."
        }
```

Reply:
Is this a requirement/standard somewhere?

We'll take it into consideration - but to change this now would be a breaking change and there was no indication that this was a standard so it's been like this for as long as the extension has existed.